### PR TITLE
Working on #2972 Template Conversion to Twig Format (IP Addresses)

### PR DESCRIPTION
**Pull request by PaulBender**
#2972 for the moderation.php IP Address templates.

Also fixes a minor issue with IP addresses on the postbit.

### CODE REVIEW

File: inc/views/base/moderation/getip.twig [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            <td class="trow1"><strong>{{ lang.hostname }}<br /><span class="smalltext">{{ lang.if_resolvable }}</span></strong></td>
+            <td class="trow2">{{ post.hostname }}</td>
+        </tr>
+        {% if mybb.usergroup.canmodcp and mybb.usergroup.canuseipsearch %}
+            <tr>
+                <td class="trow2"><strong>{{ lang.mod_options }}</strong></td>
+                <td class="trow2">
+                    <a href="modcp.php?action=ipsearch&amp;ipaddress={{ post.ipaddress }}&amp;search_users=1">{{ lang.search_ip_users }}</a><br />
+                    <a href="modcp.php?action=ipsearch&amp;ipaddress={{ post.ipaddress }}&amp;search_posts=1">{{ lang.search_ip_posts }}</a><br />
+                    <a href="modcp.php?action=iplookup&amp;ipaddress={{ post.ipaddress }}" onclick="MyBB.popupWindow('/modcp.php?action=iplookup&ipaddress={{ post.ipaddress }}&modal=1'); return false;">{{ lang.info_on_ip }}</a>
```
Comment (euantorano) [inline inc/views/base/moderation/getip.twig]: The `&`s here should be `&amp;`.

File: inc/views/base/moderation/getip_modal.twig [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                <td class="trow1"><strong>{{ lang.hostname }}<br /><span class="smalltext">{{ lang.if_resolvable }}</span></strong></td>
+                <td class="trow2">{{ post.hostname }}</td>
+            </tr>
+            {% if mybb.usergroup.canmodcp and mybb.usergroup.canuseipsearch %}
+                <tr>
+                    <td class="trow2"><strong>{{ lang.mod_options }}</strong></td>
+                    <td class="trow2">
+                        <a href="modcp.php?action=ipsearch&amp;ipaddress={{ post.ipaddress }}&amp;search_users=1">{{ lang.search_ip_users }}</a><br />
+                        <a href="modcp.php?action=ipsearch&amp;ipaddress={{ post.ipaddress }}&amp;search_posts=1">{{ lang.search_ip_posts }}</a><br />
+                        <a href="modcp.php?action=iplookup&amp;ipaddress={{ post.ipaddress }}" onclick="MyBB.popupWindow('/modcp.php?action=iplookup&ipaddress={{ post.ipaddress }}&modal=1'); return false;">{{ lang.info_on_ip }}</a>
```
Comment (euantorano) [inline inc/views/base/moderation/getip_modal.twig]: Same again here: `&amp;`

File: inc/views/base/moderation/getpmip.twig [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            <td class="trow1"><strong>{{ lang.hostname }}<br /><span class="smalltext">{{ lang.if_resolvable }}</span></strong></td>
+            <td class="trow2">{{ pm.hostname }}</td>
+        </tr>
+        {% if mybb.usergroup.canmodcp and mybb.usergroup.canuseipsearch %}
+            <tr>
+                <td class="trow2"><strong>{{ lang.mod_options }}</strong></td>
+                <td class="trow2">
+                    <a href="modcp.php?action=ipsearch&amp;ipaddress={{ pm.ipaddress }}&amp;search_users=1">{{ lang.search_ip_users }}</a><br />
+                    <a href="modcp.php?action=ipsearch&amp;ipaddress={{ pm.ipaddress }}&amp;search_posts=1">{{ lang.search_ip_posts }}</a><br />
+                    <a href="modcp.php?action=iplookup&amp;ipaddress={{ pm.ipaddress }}" onclick="MyBB.popupWindow('/modcp.php?action=iplookup&ipaddress={{ pm.ipaddress }}&modal=1'); return false;">{{ lang.info_on_ip }}</a>
```
Comment (euantorano) [inline inc/views/base/moderation/getpmip.twig]: And again.

**Comment by PaulBender** [comment]
Fixed

**Pull request merged**